Lib/opcode.py

```diff
@@ -295,12 +267,29 @@ def pseudo_op(name, op, real_ops):
 
 MAX_PSEUDO_OPCODE = MIN_PSEUDO_OPCODE + len(_pseudo_ops) - 1
 
-del def_op, name_op, jrel_op, jabs_op, pseudo_op
+del def_op, pseudo_op
 
 opname = ['<%r>' % (op,) for op in range(MAX_PSEUDO_OPCODE + 1)]
 for op, i in opmap.items():
     opname[i] = op
 
+# _opcode may not be ready during early stages of the build
+try:
+    import _opcode
+except ImportError:
+    hasarg = []
+    hasconst = []
+    hasname = []
+    hasjrel = []
+    hasjabs = []
+    haslocal = []
+else:
+    hasarg = [op for op in opmap.values() if _opcode.has_arg(op)]
+    hasconst = [op for op in opmap.values() if _opcode.has_const(op)]
+    hasname = [op for op in opmap.values() if _opcode.has_name(op)]
+    hasjrel = [op for op in opmap.values() if _opcode.has_jump(op)]
+    hasjabs = []
```

Comment:
Do we want to keep the hasjabs list now that all jumps are relative? Do we want to rename hasjrel to hasjump?

Note that these lists are documented: https://docs.python.org/3/library/dis.html#opcode-collections

Reply:
Maybe we can introduce hasjump but keep the other two for backward compatibility, and (soft?) deprecate them in the docs only. @carljm